sys/vane/clay.hoon

```diff
@@ -2372,9 +2372,17 @@
     ++  read-p-in
       |=  {pax/path pes/regs}
       ^-  dict
-      =+  rul=(~(get by pes) pax)
-      ?^  rul  [pax u.rul]
-      ?~  pax  [/ %white ~]
+      =/  rul=(unit rule)  (~(get by pes) pax)
+      ?^  rul
+        :+  pax  mod.u.rul
+        %-  ~(rep in who.u.rul)
+        |=  {w/whom out/(pair (set @p) (map @ta crew))}
```

Comment:
Might want to use `ship` in place of `@p`, like the structure definition in zuse does.